app/models/thesis.rb

```diff
@@ -128,8 +128,17 @@ class Thesis < ApplicationRecord
   scope :exported_to_proquest, -> { where(proquest_exported: ['Partial harvest', 'Full harvest']) }
   scope :not_exported_to_proquest, -> { where(proquest_exported: 'Not exported') }
   scope :published, -> { where(publication_status: 'Published') }
-  scope :partial_proquest_export, -> { published.doctoral.not_exported_to_proquest.not_consented_to_proquest }
-  scope :full_proquest_export, -> { published.advanced_degree.not_exported_to_proquest.consented_to_proquest }
+
+  # Only theses with from the 9/2022 degree period and onward should be exported to ProQuest.
+  scope :proquest_degree_period, -> { where('grad_date >= ?', Date.parse('1-9-2022')) }
```

Comment:
To help my American brain, would you mind swapping this format to `Date.parse('September 2022')` though ought this changeset? What you have here is 100% correct and accurate but it also hurts a bit to have to parse the parsing.

Reply:
Oh, I didn't think of that. Good call.